js/game.js

```diff
@@ -0,0 +1,10 @@
+class Game {/**Evil comments with my freaky peasant language skills*/
```

Comment:
Kā šis komentārs iederas kontekstā?

Reply:
Atvainojos par pārprotamo tekstu, es mācos programmēt un lietot angļu valodu. Saprotu, ka šis komentārs izskatās kā ar krāsas baloniņu uzpūsts pincītis uz uzņēmuma parādes durvīm.